index.js

```diff
@@ -17,31 +17,41 @@ module.exports = postcss.plugin("postcss-calc", function(options) {
   var warnWhenCannotResolve = options.warnWhenCannotResolve
 
   return function(style, result) {
+    function transformValue(node, property) {
+      var value = node[property]
 
-    style.walkDecls(function transformDecl(decl) {
-      if (!decl.value || decl.value.indexOf("calc(") === -1) {
+      if (!value || !CONTAINS_CALC.test(value)) {
```

Comment:
maybe we should stick to a simple indexOf for everything instead of a regexp? (for perf)